src/app/components/FormSelect/index.tsx

```diff
@@ -27,6 +27,7 @@ interface Props {
   onChange: (customer: SelectItem) => void;
   inputFocus?: boolean;
   isItemDisabled?: string;
+  dataActionId?: string;
 }
 
 export function FormSelect(props: Props) {
```

Comment:
I wouldn't mention it because we're gonna rewrite the dapp + it's not very significant but you're gonna need to touch this code anyway :D Rewrite it pls to `React.FC` and rename `Props` appropriately.